translators/test/snapshot/IndicesParameters_Datatypes.idr

```diff
@@ -2,7 +2,11 @@ module Main
 
 
 data D : (p1 : Type) -> (p2 : Type) -> (p3 : Type) -> (p4 : Type) -> (p5 : Type) -> Nat -> Nat -> Nat -> Nat -> Nat -> Type where
- C : p1 -> p2 -> p3 -> p4 -> p5 -> {X1, X2, X3, X4, X5 : Nat} -> D p1 p2 p3 p4 p5 X1 X2 X3 X4 X5
+ C : p1
+  -> p2
+  -> p3
+  -> p4
+  -> p5 -> {X1, X2, X3, X4, X5 : Nat} -> D p1 p2 p3 p4 p5 X1 X2 X3 X4 X5
```

Comment:
Looks like there is some bad break hinting here: this is always a bit fiddly to get right. We should probably just let this slide for now and port all the others: some common patterns are going to emerge, and we should be able to design more layout-sensitive combinators.

Reply:
exactly my thinking, which is why I let this bad break slip through.